tests/functional/u/unused/unused_import.py

```diff
@@ -98,8 +99,10 @@ def blop(self):
     import zoneinfo
 
 
-class WithMetaclass(metaclass=ABCMeta):
```

Comment:
This test was altered in #7320 to remove six, but that made it unable to target the astroid behavior. The original test in #7153 didn't quite test this correctly: it confounded things by adding `, object`.